modules/yarpc/yarpc_options.go

```diff
@@ -1,47 +0,0 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
yarpc_options.go is so small (and logically connected to `Module`) that in this case it makes sense to have the code right next to `Module` and newModule. It makes more sense for a developer in my opinion to immediately see these 20ish lines of code next to where `ModuleOptions` are used.